<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddItemWindow</class>
 <widget class="QDialog" name="AddItemWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>&lt;h2&gt;Add New&lt;/h2&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameInput"/>
   </item>
   <item>
    <widget class="QPushButton" name="createBtn">
     <property name="text">
      <string>Create New </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
